daemon/src/monitor.rs

```diff
@@ -82,14 +82,20 @@ where
                     actor.cfds.insert(cfd.order.id, params.clone());
                     actor.monitor_all(&params, cfd.order.id);
                 }
-                CfdState::Open { dlc, .. } | CfdState::PendingCommit { dlc, .. } => {
+                CfdState::Open { dlc, collaborative_close, .. } | CfdState::PendingCommit { dlc, collaborative_close, .. } => {
```

Comment:
:upside_down_face: We could add a method `collaborative_close` on `CfdState` that returns an `Option` of the `collaborative_close` field and check that after the `match` statement. We could reduce some code duplication that way.